tools/klee/main.cpp

```diff
@@ -220,7 +220,8 @@ class KleeHandler : public InterpreterHandler {
 
   SmallString<128> m_outputDirectory;
 
-  unsigned m_testIndex;  // number of tests written so far
+  unsigned m_totalTests;    // number of tests found so far
+  unsigned m_successTests;  // number of successful tests written so far
```

Comment:
I don't like the name `m_succcessTests`. It's ambiguous what "success" means. Perhaps it should be something like called `m_numGeneratedTests`  and for consistency `m_totalTests` could be `m_numTotalTests`?